### PR TITLE
Adding the ability to set the default HTTP timeout

### PR DESCRIPTION
**Pull request by arschles**
https://github.com/kedacore/keda/pull/1251 adds a new environment variable to allow users to set the default HTTP timeout in the system. This exposes that setting in the helm chart.

### CODE REVIEW

**Comment by arschles** [comment]
The failure in https://github.com/kedacore/charts/pull/90/checks?check_run_id=1311498573 is this: 

```shell
Error:  templates/21-metrics-clusterrolebinding.yaml: object name does not conform to Kubernetes naming requirements: "keda-operator:system:auth-delegator": invalid metadata name, must match regex ^(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])+$ and the length must not longer than 253
```

I didn't touch that file so I think it's unrelated to my change, but I'm happy to fix it if you like.

**Comment by tomkerkhove** [comment]
> The failure in [#90 (checks)](https://github.com/kedacore/charts/pull/90/checks?check_run_id=1311498573) is this:
> 
> ```shell
> Error:  templates/21-metrics-clusterrolebinding.yaml: object name does not conform to Kubernetes naming requirements: "keda-operator:system:auth-delegator": invalid metadata name, must match regex ^(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])+$ and the length must not longer than 253
> ```
> 
> I didn't touch that file so I think it's unrelated to my change, but I'm happy to fix it if you like.

This is ok, you can ignore that one.

**Review by tomkerkhove — CHANGES_REQUESTED** [review]
LGTM, but can you update the README as well please?

**Comment by tomkerkhove** [comment]
Will hold off until the PR gets merged as well.

**Comment by arschles** [comment]
@tomkerkhove sorry for the delay. I made this update in conjunction with https://github.com/kedacore/keda/pull/1251. I'm not sure what order they should be merged, but please let me know if you'd like help with it.

File: keda/values.yaml
Comment (tomkerkhove) [inline keda/values.yaml:128]: Maybe a nit but what if we use `http.timeout` here to align with the other conventions?

```suggestion
http:
  timeout: 3000
```

**Review by tomkerkhove — APPROVED** [review]
LGTM with a suggestion, but waiting with merging until PR gets merged

**Comment by tomkerkhove** [comment]
Any update on this @arschles ?

**Comment by zroubalik** [comment]
@arschles could you update this pls?

**Comment by arschles** [comment]
forgot about this one, sorry folks! I'll update this tomorrow first thing

**Comment by arschles** [comment]
@tomkerkhove @zroubalik resolved the conflicts and made the `httpTimeout` ==> `http.timeout` change that you requested @tomkerkhove .

Sorry again about the delay. I didn't save this repository in my notifications presets - since fixed!

**Review by tomkerkhove — APPROVED** [review]
LGTM

**Comment by tomkerkhove** [comment]
Thanks @arschles!

**Comment by tomkerkhove** [comment]
Can you fix the deploy issue please?

**Comment by arschles** [comment]
@tomkerkhove my bad - fixed and merging now.

**Comment by tomkerkhove** [comment]
Thank you!

**Pull request merged**
